src/config.js

```diff
@@ -143,14 +144,14 @@ global.config = {
     structureAvoid: 0xFF,
     creepAvoid: 0xFF,
     wallThickness: 1,
-    version: 16
+    version: 16,
   },
 
   mineral: {
     enabled: false,
     storage: 100000,
     minAmount: 5000,
-    minAmountForMarket: 100000
+    minAmountForMarket: 100000,
```

Comment:
Okay I will review codeclimate settings, in the meantime, check my other suggestion